### PR TITLE
Scheduling: setTimeout and setInterval

### PR DESCRIPTION
**Pull request by mahdiHashemi14**
Line 77 and 226 need more attention  
Please review and comment/commit if anything is wrong

### CODE REVIEW

File: 1-js/06-advanced-functions/08-settimeout-setinterval/1-output-numbers-100ms/task.md [hunk 1]
```diff
@@ -2,11 +2,11 @@ importance: 5
 
 ---
 
-# Output every second
+# هر ثانیه را خروجی بگیرید
```
Comment (mahdyar) [inline 1-js/06-advanced-functions/08-settimeout-setinterval/1-output-numbers-100ms/task.md:5]:
```suggestion
# در هر ثانیه خروجی بگیرید
```

File: 1-js/06-advanced-functions/08-settimeout-setinterval/4-settimeout-result/task.md [hunk 2]
```diff
@@ -2,25 +2,25 @@ importance: 5
...
-1. After the loop.
-2. Before the loop.
-3. In the beginning of the loop.
+1. بعد از حلقه.
+2. قبل از حلقه.
+3. در آغاز حلقه.
 
 
-What is `alert` going to show?
+`alert` قرار است چه چیزی را نمایش دهد؟
```
Comment (mahdyar) [inline 1-js/06-advanced-functions/08-settimeout-setinterval/4-settimeout-result/task.md:16]:
```suggestion
قرار است `alert` چه چیزی را نمایش دهد؟
```

**Review by mahdyar — CHANGES_REQUESTED** [review]

**Comment by mahdiHashemi14** [comment]
Thank you @mahdyar 
/done

**Review by mahdyar — APPROVED** [review]
Thank you, @mahdiHashemi14!

**Pull request merged**